commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPFriendlyURLEntryLocalServiceImpl.java

```diff
@@ -322,13 +322,8 @@ protected void validate(
 		}
 
 		if (classPK > 0) {
-			CPFriendlyURLEntry cpFriendlyURLEntry =
-				cpFriendlyURLEntryPersistence.fetchByG_C_C_L_U(
-					groupId, classNameId, classPK, languageId, urlTitle);
-
-			if (cpFriendlyURLEntry != null) {
-				return;
-			}
+			cpFriendlyURLEntryPersistence.fetchByG_C_C_L_U(
+				groupId, classNameId, classPK, languageId, urlTitle);
```

Comment:
@marco-leo what are we doing here? This looks like a useless fetch -- the new version is functionally the same as the old one, just more obviously useless.